lms/templates/edxnotes/edxnotes.html

```diff
@@ -3,72 +3,97 @@
-            % else:
-                <section class="is-empty">
-                    <article>
-                        <h2>${_('You have not made any notes in this course yet.')}</h2>
-                        <p>${_('Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')}</p>
+        % else:
+        <section class="is-empty">
+          <article>
+            <h2>${_('You have not made any notes in this course yet.')}</h2>
 
-                        <h2>${_('Other Students In This Course Are Using Notes To:')}</h2>
-                        <ul>
-                            <li>${_('Add study notes to course content for exams and homework.')}</li>
-                            <li>${_('Highlight important concepts for later coursework or future courses')}</li>
-                            <li>${_('Donec id elit non mi porta gravida et eget metus.')}</li>
-                            <li>${_('Praesent commodo cursus magna, vel scelerisque nisl consectetur et.')}</li>
-                        </ul>
+            <p>${_('Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.')}</p>
 
-                    % if position is not None:
-                        <div class="student-notes-cta">
-                            <h3 class="sr">${_('Start creating notes')}</h3>
-                            <p>${_('Get started by making a note in something you just read, like {section_link}.').format(
-                                section_link='<a href="{url}">{section_name}</a>'.format(
-                                    url=position['url'],
-                                    section_name=position['display_name'],
-                                )
-                            )}</p>
-                        </div>
-                    % endif
-                    </article>
-                </section>
+            <h2>${_('Other Students In This Course Are Using Notes To:')}</h2>
+            <ul>
+              <li>${_('Add study notes to course content for exams and homework.')}</li>
+              <li>${_('Highlight important concepts for later coursework or future courses')}</li>
+              <li>${_('Donec id elit non mi porta gravida et eget metus.')}</li>
```

Comment:
@explorerleslie and @srpearce, this placeholder copy is meant for students who haven't created any notes yet in a course. Any chance either of you can help replace the lorem ipsum @frrrances noted?